src/panels/config/zha/ha-config-zha.ts

```diff
@@ -34,18 +51,68 @@ export class HaConfigZha extends LitElement {
               icon="hass:arrow-left"
               @click="${this._onBackTapped}"
             ></paper-icon-button>
+            <div main-title>Zigbee Home Automation</div>
           </app-toolbar>
         </app-header>
 
         <zha-network
           id="zha-network"
-          .is-wide="${this.isWide}"
+          .isWide="${this.isWide}"
           .hass="${this.hass}"
         ></zha-network>
+
+        <zha-node
+          id="zha-node"
+          .isWide="${this.isWide}"
+          .hass="${this.hass}"
+          @zha-cluster-selected="${this._onClusterSelected}"
+          @zha-node-selected="${this._onNodeSelected}"
+          @zha-entity-selected="${this._onEntitySelected}"
+        ></zha-node>
+
+        <zha-cluster-attributes
+          id="zha-cluster-attributes"
+          .isWide="${this.isWide}"
+          .hass="${this.hass}"
+          .selectedNode="${this._selectedNode}"
+          .selectedEntity="${this._selectedEntity}"
+          .selectedCluster="${this._selectedCluster}"
+          ?hidden="${!this._selectedCluster}"
```

Comment:
Don't use `?hidden`, use a ternary to not render it if there is no selected cluster.

Reply:
Done. Just curious why? I'd like to understand the reasoning. Is it to keep it out of the dom vs hiding it or is there some other reason?

Reply:
If we create it but hide it, we're doing extra work.

Reply:
makes sense. Thanks